### ISSUE TITLE
Fix windows tests (CI)

### ISSUE THREAD

**Issue by Tapppi** [conventional-changelog/standard-version#59]
Ok, not sure how I managed to miss this before. Tests with mocked git don't work on windows due to [`mock-git`](https://github.com/stevemao/mock-git) and its dep [`mock-bin`](https://github.com/stevemao/mock-bin):
- [x] `mock-bin` creates an executable script with a shebang, which windows does not support, PR: (https://github.com/stevemao/mock-bin/pull/3)
- [x] `mock-git` doesn't escape slashes, PR: (https://github.com/stevemao/mock-git/pull/3)
- [x] `mock-git` has no version range in its `mock-bin` dependency thus not getting the windows fix when it lands, issue: (https://github.com/stevemao/mock-git/issues/5)

Both of these packages will need updates to make the mock tests work. I have created PRs in the respective repos and verified that standard-version tests are fixed with them. This issue is to track the progress of those fixes.

**Comment by Tapppi** [comment]
All issues resolved, just waiting on a `mock-git` release.

**Comment by bcoe** [comment]
@Tapppi digging into the AppVeyor stuff, I'm pretty stumped. A few things I noticed:

* there's a CI variable set to `true`, I wonder if this is somehow getting picked up in the formatting string.
* running "%cd --date=iso" causes an exception, unknown date format ISO. As %ci is supposed to be iso format itself, is this causing the issue?

CC: @nexdrew

**Comment by Tapppi** [comment]
@bcoe, I'm gonna sum the CI shenanigans here since it's been all over the place now. @nexdrew 's [comment in the other PR](https://github.com/conventional-changelog/standard-version/pull/62#issuecomment-228083858) had everything right, so:

1. The whole Appveyor test is broken, but not affecting use or tests on Windows locally. @bcoe, as you point out, the `CI` env var is exactly the thing breaking the build, but that cannot be removed, and as such the formatting string (which has `%ci%` in it causing it to be picked up) needs to be changed. Explanation can be found in @nexdrew 's comment and above it in mine. I'll create PRs in `conventional-changelog-core` tonight, unless someone can do it before me, for 
    * Appveyor CI config
    * Adding a space after %ci in the format string to fix this
2. The `mock-git` tests dont work, but will be fixed as soon as @stevemao releases a new version of `mock-git` with the fixes outlined in this issue's OP.

@bcoe @nexdrew @stevemao does this clear this stuff up finally? That should be a good push forward IMO.

**Comment by stevemao** [comment]
`mock-git` should work on Windows now.

**Comment by Tapppi** [comment]
Closed #57 and renamed this to reflect the fact that, at least IMO, this is the best place to track the Appveyor CI issues for now.

**Comment by Tapppi** [comment]
Opened PRs for appveyor config on conventional-changelog-core and git-raw-commits. Also opened PR in conventional-changelog/git-raw-commits: https://github.com/conventional-changelog/git-raw-commits/pull/10. It's the most logical place, since it will actually fix all percent sign variable expansion problems by escaping them on windows.

**Comment by bcoe** [comment]
@Tapppi this agrees with what I was seeing, moving forward with fixes seems smart to me ... I don't think any of us want to spin up Windows VMs too frequently, having AppVeyor in place to stop regressions will be nice.

**Comment by Tapppi** [comment]
So unfortunately my previous fix to `git-raw-commits` was a non-fix.. [Opened a new PR with an actual fix and tests to confirm it](https://github.com/conventional-changelog/git-raw-commits/pull/11)...

**Comment by Tapppi** [comment]
All but node 0.10 work now, with `git-raw-commits v1.1.2`. Don't have time right now to delve into that..

https://ci.appveyor.com/project/Tapppi/standard-version/build/29

**Comment by bcoe** [comment]
@Tapppi I'm pretty comfortable with removing `Node 0.10` from Windows integration tests, if you don't have time to dig into this further ... gets us to a much better place than where we were.

**Comment by Tapppi** [comment]
Yeah I think that's the best move forwards.

**Comment by bcoe** [comment]
@Tapppi great work!

**Comment by stevemao** [comment]
Thanks @tapppi!
